cedar-policy/src/api/err.rs

```diff
@@ -134,14 +134,7 @@ pub enum ReauthorizationError {
     /// A policy set error was encountered
     #[error(transparent)]
     #[diagnostic(transparent)]
-    PolicySet(#[from] PolicySetError),
-}
-
-#[doc(hidden)]
-impl From<cedar_policy_core::ast::PolicySetError> for ReauthorizationError {
-    fn from(e: cedar_policy_core::ast::PolicySetError) -> Self {
-        Self::PolicySet(e.into())
-    }
+    Reauthorization(#[from] cedar_policy_core::authorizer::ReauthorizationError),
```

Comment:
Since `cedar_policy::ReauthorizationError` is a public enum, this contained struct needs to be either re-exported or wrapped.  (In a public enum, all data contained in each variant is always `pub`.)

Reply:
Pushed a fix.